run_tuxemon.py

```diff
@@ -52,4 +55,16 @@
         config.skip_titlescreen = True
         config.splash = False
 
-    main.main(load_slot=args.slot)
+    try:
+        main.main(load_slot=args.slot)
+    except Exception as ex:
+        from pygame import quit as pg_quit
+        pg_quit()
+        
+        logging.error(ex, exc_info=True)
+        if ex != KeyboardInterrupt and ex != EOFError:
+            #if input("Send logs? (Y/N)\n> ").upper() == "Y":
```

Comment:
is this needed?

Reply:
The logging.error part is for printing the traceback, so it isn't "eaten" by the except function.

The commented if is not needed, I'll remove it later.